### PR TITLE
70th anniversary styles no highlight

### PR DESCRIPTION
**Pull request by fjp2cornell**

### CODE REVIEW

File: docroot/sites/all/themes/ilr_theme/js/ilr_theme.js [hunk 1]
```diff
@@ -162,23 +162,24 @@
           if (mobileNavActive()) {
             $('.subsite-header').removeAttr('style');
           }
```
Comment (a-fro) [inline docroot/sites/all/themes/ilr_theme/js/ilr_theme.js]: Please remove this temp fix

File: docroot/sites/all/themes/ilr_theme/js/ilr_theme.js [hunk 2]
```diff
@@ -203,7 +204,8 @@
       var getCurrentOffset = function() {
         if (widthChanged) {
           if (subsite && !mobileNavActive()) {
```
Comment (a-fro) [inline docroot/sites/all/themes/ilr_theme/js/ilr_theme.js]: Please remove and/or fix.

**Comment by a-fro** [comment]
Hi John. Thanks for your work on this. The next step is to begin refactoring the _ilr-at-70-homepage-layout.scss file. Please make sure that anything in the layout file is layout-specific. Ultimately, this file should be pretty small and succinct. All of the component logic should move to new component files that you create, ie. reflections, timeline, list-events, add-reflection-form, etc. 

Once you have the components isolated, think about ways you can reduce specificity and nesting, and create re-usable extendable components where appropriate (such as button styles). In particular, we should begin striving to use element id's as little as possible. This is an ongoing goal of the refactoring I'd like us to do, as opposed to just in this case. Please let me know if you have questions.

**Comment by a-fro** [comment]
Also, I cherry-picked that refactoring change that was overwritten during the rebase, so you will need to pull this down from github to update your local branch. It's 400k, rather than 1.1 MB.

**Comment by fjp2cornell** [comment]
Aaron, can you review the fcbb43a commit where the reflection styles have been refactored? Thanks.

**Comment by fjp2cornell** [comment]
Aaron, I did another refactor of the styles this morning. Could you please review them for me? Based on a conversation I had with David yesterday, I have created a new sass file called _ilr_at_70.scss that contains styles specific to the ILR at 70 site. This solution is similar to what you have done with _homepage.scss. I have also refactored list events, layout, timeline and the add reflections form.  The reflections form file has styles for layout in it and wondering if they should be removed. Thanks.

**Pull request closed without merging**